WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeErrorFragment.kt

```diff
@@ -0,0 +1,64 @@
+package org.wordpress.android.ui.publicize
+
+import android.app.AlertDialog
+import android.app.Dialog
+import android.os.Bundle
+import android.view.ContextThemeWrapper
+import androidx.annotation.StringRes
+import androidx.fragment.app.DialogFragment
+import org.wordpress.android.R
+import org.wordpress.android.R.style
+import org.wordpress.android.ui.WPWebViewActivity
+import org.wordpress.android.ui.publicize.PublicizeConstants.PUBLICIZE_FACEBOOK_SHARING_SUPPORT_LINK
+import java.lang.NullPointerException
+
+class PublicizeErrorFragment : DialogFragment() {
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        arguments?.let {
+            it.getInt(REASON_RES_ID).let { reasonResId ->
+                // the custom dialog behavior here is specific to a user who is trying to share to Facebook
+                // but does not have any available Facebook Pages.
+                if (reasonResId == R.string.sharing_facebook_account_must_have_pages) {
```

Comment:
I flipped the condition here to `!=` to test the other dialog to verify it works as expected.